tests/test_xgcm_util.py

```diff
@@ -7,9 +7,12 @@
 
 
 @pytest.mark.parametrize('file', ['tend_zint_100m_Fe.nc', 'g.e20.G.TL319_t13.control.001_hfreq.nc'])
-def test_xgcm_grid(file):
+def test_to_xgcm_grid_dataset(file):
     filepath = DATASETS.fetch(file)
     ds = xr.open_dataset(filepath)
-    grid = pop_tools.get_xgcm_grid(ds, metrics=None)
+    grid, ds_new = pop_tools.to_xgcm_grid_dataset(ds, metrics=None)
     assert isinstance(grid, xgcm.Grid)
     assert set(['X', 'Y', 'Z']) == set(grid.axes.keys())
+    new_spatial_coords = set(['nlon_u', 'nlat_u', 'nlon_t', 'nlat_t'])
+    for coord in new_spatial_coords:
+        assert coord in ds_new.coords
```

Comment:
let's also assert that none of these new coords are in the original `ds`

Reply:
Awesome. This should be resolved by the latest commit.